clientGig/Components/ClientDashboard/Performers/PerformerView.js

```diff
@@ -11,6 +12,12 @@ class PerformerView extends React.Component {
   }
 
   componentDidMount() {
+    console.log(this.props.applicants[0]);
+    Axios.get(
+      `http://localhost:8000/api/client/performerinfo${this.props.applicants[0]}`

```

Comment:
Change url to be better practice: http://localhost:8000/api/client/performerinfo/${this.props.applicants[0]}